<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>93</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SoundStream demo</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="QPushButton" name="btnPlay">
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnPause">
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnStop">
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
